.github/workflows/upgrade-python-requirements.yml

```diff
@@ -88,10 +89,11 @@ jobs:
           USER_REVIEWERS: ${{ inputs.user_reviewers }}
           TEAM_REVIEWERS: ${{ inputs.team_reviewers }}
         run: |
-          cd $GITHUB_WORKSPACE/testeng-ci
-          python -m jenkins.pull_request_creator --repo-root=$GITHUB_WORKSPACE \
+          cd $GITHUB_WORKSPACE/repo-tools
```

Comment:
I don't think you need to CD into this repo now that the binary is on the path do you?